qutebrowser/browser/downloads.py

```diff
@@ -131,7 +131,11 @@ def create_full_filename(basename, filename):
     encoding = sys.getfilesystemencoding()
     filename = utils.force_encoding(filename, encoding)
     basename = utils.force_encoding(basename, encoding)
-    if os.path.isabs(filename) and os.path.isdir(filename):
+
+    if filename.endswith('/'):
```

Comment:
How does qutebrowser currently deal with directory separators? Most "unix-y programs" accept `/` on Unix, and both `/` and `\` on Windows. So just checking `os.sep` doesn't seem too great either (but if that's what the rest of qutebrowser does, then that's okay, I guess).